controllers/static/static_controller.go

```diff
@@ -0,0 +1,103 @@
+package static
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+	"github.com/netobserv/network-observability-operator/controllers/consoleplugin"
+	"github.com/netobserv/network-observability-operator/controllers/reconcilers"
+	"github.com/netobserv/network-observability-operator/pkg/helper"
+	"github.com/netobserv/network-observability-operator/pkg/manager"
+	"github.com/netobserv/network-observability-operator/pkg/manager/status"
+)
+
+const (
+	initReconcileAttempts = 5
+)
+
+type Reconciler struct {
+	client.Client
+	mgr    *manager.Manager
+	status status.Instance
+}
+
+func Start(ctx context.Context, mgr *manager.Manager) error {
+	log := log.FromContext(ctx)
+	log.Info("Starting Static controller")
+	r := Reconciler{
+		Client: mgr.Client,
+		mgr:    mgr,
+		status: mgr.Status.ForComponent(status.StaticPlugin),
+	}
+
+	// force reconcile at startup
+	go r.InitReconcile(ctx)
+
+	return ctrl.NewControllerManagedBy(mgr).
+		For(&flowslatest.FlowCollector{}, reconcilers.IgnoreStatusChange).
+		Named("staticPlugin").
+		Complete(&r)
+}
+
+func (r *Reconciler) InitReconcile(ctx context.Context) {
+	log := log.FromContext(ctx)
+	log.Info("Initializing resources...")
+
+	for attempt := range initReconcileAttempts {
```

Comment:
btw, what happens if all 5 attempts fail? The static plugin wouldn't deploy, but the rest would work normally? Or does it may the controller CLBO or so ?

Reply:
If the attempts fails, the static plugin will not be there at controller startup.
As soon as a reconcile loop is triggered, it will appears (ie creating a FlowCollector for example)